logger.go

```diff
@@ -115,10 +122,15 @@ func (p *LogFormatterParams) ResetColor() string {
 	return reset
 }
 
+// IsOutputColor indicates whether can colors be outputted to the log.
+func (p *LogFormatterParams) IsOutputColor() bool {
+	return consoleColorMode == forceColor || (consoleColorMode == autoColor && p.IsTerm)
```

Comment:
`IsTerm bool` need to export?

Reply:
I suppose so. Done. https://github.com/gin-gonic/gin/pull/1802/commits/d6440ae2499376b7adaa3ae24ccdb10faaa241a2

Reply:
thanks!